operations.js

```diff
@@ -212,15 +212,26 @@ Operations.prototype.getPending = function getPending() {
     return self.pending;
 };
 
+Operations.prototype.getOutTombstone = function getOutTombstone(id) {
+    var self = this;
+
+    var tombstone = self.requests.out[id] || null;
+    if (tombstone && !tombstone.isTombstone) {
```

Comment:
would be better to call this `var op` then since we're not sure it's a tombstone until after this check